Source/Meadow.ProjectLab.Samples.sln

```diff
@@ -88,6 +88,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MicroLayoutMenu", "MicroLay
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Graphics.MicroLayout", "..\..\Meadow.Foundation\Source\Meadow.Foundation.Libraries_and_Frameworks\Graphics.MicroLayout\Driver\Graphics.MicroLayout.csproj", "{FE4F6EC2-54EC-40E5-82C0-B622F5EDD917}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Modbus Server", "Modbus Server", "{91CF7813-4612-402E-8E39-78745A2D42B1}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MeadowModbusServer", "ModbusServer\MeadowModbusServer\MeadowModbusServer.csproj", "{ED974FE8-DC3C-4EF5-9F1A-62E045379778}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ModbusDesktopClient", "ModbusServer\ModbusDesktopClient\ModbusDesktopClient.csproj", "{C0AE9661-3CA2-4F8B-8092-C3C7C52A77AE}"

```

Comment:
Can you rename this project to `DesktopModbusClient`? my convention is starting where this project intends to run (`MeadowSomething` runs on Meadow, `DesktopSomething` runs on Desktop, `MobileSomething` runs on a Mobile device)